api/test/metrics/noop_metrics_test.cc

```diff
@@ -43,7 +43,7 @@ TEST(NoopMeter, RecordBatch)
   std::unique_ptr<Meter> m{std::unique_ptr<Meter>(new NoopMeter{})};
 
   std::map<std::string, std::string> labels = {{"Key", "Value"}};
-  auto labelkv = opentelemetry::trace::KeyValueIterableView<decltype(labels)>{labels};
+  auto labelkv = opentelemetry::common::KeyValueIterableView<decltype(labels)>{labels};
```

Comment:
Not related to this PR, but seems we need to be consistent with the scope resolution usage - either use `opentelemetry::common::KeyValueIterableView` or `common::KeyValueIterableView`. 

Nothing wrong with this PR thought. This needs to be fixed as separate PR for all such entities.

Reply:
Thanks for the feedback! I've filed an issue for this here: #368